wp-admin/js/widgets/media-widgets.js

```diff
@@ -182,15 +182,16 @@ wp.mediaWidgets = ( function( $ ) {
 						}
 
 						embedLinkView.dfd = $.ajax({
-							url: 'https://noembed.com/embed', // @todo Replace with core proxy endpoint once committed.
+							url: wp.media.view.settings.oEmbedProxyUrl,
 							data: {
 								url: embedLinkView.model.get( 'url' ),
 								maxwidth: embedLinkView.model.get( 'width' ),
-								maxheight: embedLinkView.model.get( 'height' )
+								maxheight: embedLinkView.model.get( 'height' ),
+								_wpnonce: wp.media.view.settings.nonce.wpRestApi
```

Comment:
Since we know that we only will be wanting YouTube and Vimeo, how about adding `discover=false` param?